assignment3.rb

```diff
@@ -0,0 +1,60 @@
+# ASSIGNMENT 3
+# Create a calculator application which can perform add, sub, division and multiplication operation. Add exception handling for operations like divide which handles ZeroDivisionError exception printing some user friendly message. Add a method in this calculator which takes a number as argument and returns:
+# “positive” - if number is positive
+# “negative” - if number is negative
+# nil - if number is 0
+
+class Calculator
+   def initialize(a,b)
+      @a = a
+      @b = b
+   end
+   def add
+     c = @a + @b
+     puts "addition of #{@a} and #{@b} is #{c}","\n"
+   end
+   def sub
+     c = @a - @b
+     puts "substraction of #{@a} and #{@b} is #{c}","\n"
+   end
+   def mul
+     c = @a / @b
```

Comment:
@monalis-cuelogic : Are you sure **multiplication** operation is being performed here ?